pkg/listener/listener.go

```diff
@@ -65,6 +65,7 @@ func (l *StatsDUDPListener) HandlePacket(packet []byte) {
 	l.UDPPackets.Inc()
 	lines := strings.Split(string(packet), "\n")
 	for _, line := range lines {
+		level.Debug(l.Logger).Log("msg", "Incoming line", "line", line)
```

Comment:
Let's also log which message was received:
```suggestion
		level.Debug(l.Logger).Log("msg", "Incoming line", "proto", "udp", "line", line)
```